### DISCUSSION TITLE
what should/will happen when register a name more than once

### DISCUSSION THREAD

**Discussion by mh-alahdadian** [react-hook-form/react-hook-form#2066] [General]
I was thinking about what would be happen when registering a gilled name more than once 
does this different about how we register it? registering two controller with same name or manual register it on useEffect or register it with ref?

will these fields get synced together?

**Comment by bluebill1049** [comment]
it will get overwritten if `ref` is diff.

**Reply by mh-alahdadian** [reply]
did you tried to sync them? what is the challenge? 
I think that we should try that, what do you think?
